server/controllers/usersController.js

```diff
@@ -35,11 +35,13 @@ export default class Users {
     if (!password) {
```

Comment:
### Critical
Following Single Responsibility Principle (SRP), validations should not be littered in your controllers. It's best to abstract all validations into one container. Your controller should only do what it says (createUser). It shouldn't be bothered about validating requests. Also, basic validations like this can be handled in your user model.

Reply:
Fixed